web/packages/teleport/src/lib/tdp/client.ts

```diff
@@ -134,7 +136,11 @@ export default class Client extends EventEmitterWebAuthnSender {
     // prior to a socket 'close' event (https://stackoverflow.com/a/40084550/6277051).
     // Therefore, we can rely on our onclose handler to account for any websocket errors.
     this.socket.onerror = null;
-    this.socket.onclose = () => {
+    this.socket.onclose = ev => {
+      let message = 'session disconnected';
+      if (ev.code !== WebsocketCloseCode.NORMAL) {
+        message = `connection closed with websocket error code: ${ev.code}`;
```

Comment:
Users aren't going to have any idea what various websocket error codes mean. This seems like something that could be logged but doesn't need to be user-facing.

Reply:
Updated in 6cfb564322a1deedff79e1451b2d5982d8f92bbe